spec/sober_swag/output_object/merge_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+RSpec.describe 'merging SoberSwag output objects' do
+  let(:base) do
+    SoberSwag::OutputObject.define do
+      identifier 'MyObject'
+      field :id, primitive(:Integer)
+      field :name, primitive(:String)
+    end
+  end
+
+  context 'when merged into a blueprint' do
+    subject { example.serialize({ id: 10, name: 'Bob', items: 10 }) }
+
+    let(:example) do
+      bp = base
+      SoberSwag::OutputObject.define do
```

Comment:
Does it take the identifier from the base object if none is defined in this one?